ads/_config.js

```diff
@@ -284,6 +284,12 @@ export const adConfig = {
     prefetch: 'https://cdn.gmossp-sp.jp/ads/amp.js',
   },
 
+  gumgum: {
+    prefetch: [
+      'https://g2.gumgum.com/javascripts/ad.js',
```

Comment:
we have a render-start API that helps to provide better user experience. Please consider supporting it. [related doc](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis)
Let me know if there's any question.